src/index.js

```diff
@@ -4,17 +4,29 @@ import ColorLib from '@kurkle/color';
 function createGradient(ctx, axis, area) {
   return axis === 'y'
     ? ctx.createLinearGradient(0, area.bottom, 0, area.top)
-    : ctx.createLinearGradient(area.left, 0, area.right, 0);
+    : axis === 'r'
```

Comment:
maybe an IF or SWITCH statement could be more readable.

Reply:
I vote for IF's.